recipes/newrelic/elasticsearch/debian.yml

```diff
@@ -0,0 +1,112 @@
+    default: "localhost"
+  - name: "NR_CLI_API_PORT"
+    prompt: "Please enter your ElasticSearch port below. If none is provided, the default value: 9200, will be used."
+    default: 9200
+  - name: "NR_CLI_CONFIG_PATH"
+    prompt: "Please provide the path to ElasticSearch config file, if none is provided then the default: /etc/elasticsearch/elasticsearch.yml will be used."
+    default: "/etc/elasticsearch/elasticsearch.yml"
+  - name: "NR_CLI_API_USE_SSL"
+    prompt: "Do you want to use SSL to connect?"
+    default: false
+  - name: "NR_CLI_API_CA_BUNDLE_DIR"
+    prompt: "Please enter the directory in which the SSL certificate is located on the host. If using SSL please leave this field empty."
+  - name: "NR_CLI_API_CA_BUNDLE_FILE"
+    prompt: "Please enter the path to the SSL certificate file on the host. If using SSL please leave this field empty."
+install:
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - taks: assert_pre_req
+        - task: setup
+
+    assert_pre_req:
+      cmds:
+        - |
+          SERVICE_EXIST=$(sudo systemctl status newrelic-infra.service | grep "Active" | wc -l)
+          if [ $SERVICE_EXIST -eq 0 ]; then
+            echo "The newrelic-infra agent service is NOT installed on the host, but is required to install this integration." >> /dev/stderr
+            exit 1
+          fi
+
+    setup:
+      label: "Installing elasticsearch integration..."
+      cmds:
+        - |
+          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
+        - |
+          sudo apt-get update
+        - |
+          sudo apt-get install nri-elasticsearch -y
+        - |
+          if [ -f /etc/newrelic-infra/integrations.d/elasticsearch-config.yml ]; then
+            sudo rm /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+          fi
+
+          sudo touch /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+        - |
+          sudo tee -a /etc/newrelic-infra/integrations.d/elasticsearch-config.yml > /dev/null <<"EOT"
+          integration_name: com.newrelic.elasticsearch
+
+          instances:
+            - name: elasticsearch
+              command: all
+              arguments:
+                cluster_environment: staging
+                config_path: {{.NR_CLI_CONFIG_PATH}}
+                hostname: {{.NR_CLI_API_HOSTNAME}}
+                local_hostname: localhost
+                username: {{.NR_CLI_API_USERNAME}}
+                password: {{.NR_CLI_API_PASSWORD}}
+                port: {{.NR_CLI_API_PORT}}
+                timeout: 30
+                use_ssl: {{.NR_CLI_API_USE_SSL}}
+                ca_bundle_dir: {{.NR_CLI_API_BUNDLE_DIR}}
+                ca_bundle_file: {{.NR_CLI_API_CA_BUNDLE_FILE}}
+                indices_regex:
+                collect_indices: true
+                collect_primaries: true
+              labels:
```

Comment:
@setamyDG please remove the labels section, we don't want to setup a label of role=elasticsearch on for all installation.